lib/carbon/conf.py

```diff
@@ -246,7 +246,7 @@ def postOptions(self):
             print "Error: missing required config %s" % storage_schemas
             sys.exit(1)
 
-        if settings.CACHE_WRITE_STRATEGY not in ('sorted', 'max', 'naive'):
+        if settings.CACHE_WRITE_STRATEGY not in ('timesorted', 'sorted', 'max', 'naive'):
```

Comment:
From what I recall, random strategy has linear complexity, and so makes it terrible for high volume workloads.  Same for max strategy too.

https://github.com/graphite-project/carbon/blob/abff76a3962515003961f40f9e5fd8a3a28ed5c6/lib/carbon/tests/benchmark_cache.py#L73-L76